Can't build on macOS 10.14 Mojave

Placeholder to fix Mojave support.  When building on the latest macOS, I get the following error:

```
[ 21%] Linking C shared module ../../ladspa/pitch_scale_1193.so
[ 21%] Linking C shared module ../../ladspa/freq_tracker_1418.so
[ 21%] Generating ../../../../plugins/LadspaEffect/swh/ladspa/se4_1883.c
ld: library not found for -lfftw3f
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [plugins/ladspa/pitch_scale_1193.so] Error 1
make[1]: *** [plugins/LadspaEffect/swh/CMakeFiles/pitch_scale_1193.dir/all] Error 2
```

It may be machine-specific or Homebrew-specific (I only have one machine running Mojave).  Advice appreciated.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -71,6 +71,8 @@ OPTION(WANT_QT5		"Build with Qt5" OFF)
 
 
 IF(LMMS_BUILD_APPLE)
+	# Fix linking on 10.14+. See issue #4762 on github
+	LINK_DIRECTORIES(/usr/local/lib)
 	SET(WANT_ALSA OFF)
 	SET(WANT_PULSEAUDIO OFF)
 	SET(WANT_VST OFF)

